src/getCoinCombination.test.js

```diff
@@ -7,4 +7,45 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array with 4 elements`, () => {
+    const result = getCoinCombination();
+
+    expect(result.length).toBe(4);
+  });
+
+  it(`should return valid value for 0 cent`, () => {
+    expect(getCoinCombination(0))
+      .toStrictEqual([0, 0, 0, 0]);
+  });
+
+  it(`should return valid value for 1 cent`, () => {
```

Comment:
Consider your test description a test case summary. It should be short, clear, and informative.
should return 1 pennie when '1' is entered
should return 1 pennie and 1 nickel when '6' is entered